readme.md

```diff
@@ -309,6 +310,10 @@ namespace Azure.Service.Models
 
 </details>
 
+### Customize serialization/deserialization methods
+
+Use the [Replace any generated member](#replace-any-generated-member) approach to replace Serialize/Deserialize method with a custom implementation.
```

Comment:
Would it make sense to link to this sample? https://github.com/Azure/autorest.csharp/blob/feature/v3/test/TestProjects/TypeSchemaMapping/Customized/CustomizedModel.cs#L20

Reply:
Want to keep this document self sufficient, I tend to overcomplicate test classes by adding to many features to them. Added a sample.